extensions/sql-database-projects/src/models/project.ts

```diff
@@ -59,6 +59,15 @@ export class Project {
 			const importTarget = this.projFileXmlDoc.documentElement.getElementsByTagName(constants.Import)[i];
 			this.importedTargets.push(importTarget.getAttribute(constants.Project));
 		}
+
+		// find all SQLCMD variables to include
+		for (let i = 0; i < this.projFileXmlDoc.documentElement.getElementsByTagName(constants.SqlCmdVariable).length; i++) {
+			const sqlCmdVar = this.projFileXmlDoc.documentElement.getElementsByTagName(constants.SqlCmdVariable)[i];
+			const varName = sqlCmdVar.getAttribute(constants.Include);
+
+			const varValue = sqlCmdVar.getElementsByTagName(constants.DefaultValue)[0].childNodes[0].nodeValue;
+			this.sqlCmdVariables[varName] = varValue;
```

Comment:
tell me if I understand correctly - we are only taking the values from proj file as is?

(we should give a way to edit at deployment time... through profile or in the dialog itself like SSDT)

Reply:
Yup, it's just passthrough from the sqlproj file here to make sure it has proper interop with projects coming from SSDT.  I figured variable editing would happen when we get something like a project properties dialog going since that's the existing experience for defining and setting them.

Made #10841 to track